content/en/getting_started/synthetics/private_location.md

```diff
@@ -64,6 +67,8 @@ Your private locations test results display identically to your managed location
     ```
     
     This command starts a Docker container and prepares your private location to run tests. Datadog recommends running the container in detached mode with proper restart policy.
+
+    <div class="alert alert-info">You can use another container runtime such as Podman. For more information, see the <a href="https://docs.datadoghq.com/synthetics/private_location?tab=podman#install-your-private-location">Private Locations documentation</a>.</div>
```

Comment:
Link isn't working :(

Reply:
This link won't work until we merge this PR in!